geoportal/c2cgeoportal_geoportal/scaffolds/nondockercreate/nondocker-finalise.mk_tmpl

```diff
@@ -175,9 +175,9 @@ endif
 	$(DOCKER_RUN) cp /opt/npm-packages .
 	$(PYTHON_BIN)/python -m pip install --editable=c2cgeoportal_commons --editable=c2cgeoportal_geoportal --editable=c2cgeoportal_admin
 	$(PYTHON_BIN)/python -m pip install --editable=geoportal
-	$(PYTHON_BIN)/python -m compileall -q .build/venv 2&>/dev/null|| true
-	$(PYTHON_BIN)/python -m compileall -q c2cgeoportal_* 2&>/dev/null || true
-	$(PYTHON_BIN)/python -m compileall -q geoportal/$(PACKAGE)_geoportal -x geoportal/$(PACKAGE)_geoportal/static.* 2&>/dev/null || true
+	$(PYTHON_BIN)/python -m compileall -q .build/venv >/dev/null|| true
```

Comment:
Can you also add the missing space before `||`?

Reply:
Done